### PR TITLE
Completed body

### PR DESCRIPTION
**Pull request by allyface**

### CODE REVIEW

File: demo.html [hunk 1]
```diff
@@ -22,74 +25,118 @@ <h5 class="headings">h5. Vast Quoll</h5>
...
-      </code>
+       </code>
+     </div>
+
       <!--
-      <div class="secondary-text">
-        <h2 class="section-title"><small>Secondary Headings</small></h1>
+      <div class="secondary-headings">
+        <h2 class="section-title"><small>Secondary Headings</small></h2>
         <p>To create secondary text, add the <code class= "class-name">.small</code> class to your <code class= "element-in-text">&lt;h1&gt;</code> through <code class= "element-in-text">&lt;h6&gt;</code> tags.</p>
```
Comment (pllearns) [inline demo.html:78]: Are you going to implement this code at a later time?

File: demo.html [hunk 2]
```diff
@@ -22,74 +25,118 @@ <h5 class="headings">h5. Vast Quoll</h5>
...
-    </header>
-  </div>
+
+    <header id="body-copy">
+      <div id="body-text">
+        <h1 class="section-title">Body copy</h1>
+
+        <div class="breakdown">
+          <p>Vast CSS global default <!--
+         --><code class="element-in-text">font-size</code> is <!--
```
Comment (pllearns) [inline demo.html:95]: Are these commented arrows part of the demo?

**Review by pllearns — COMMENTED** [review]
Have you had fun with bootstrap so far? I think that the pressing things here are that you should go ahead and check on how big this demo file is becoming, but that would be a small thing as you are definitely playing with bootstrap. I also feel that by Friday you should remove the comments, unless they are essential to your retention strategy.

File: demo.html (hunk 1 above)
Comment (silentsoundz) [reply demo.html:78]: we are not sure if we are using it but don't want to delete yet.

**Pull request merged**